### PR TITLE
v1.0.11

### PR DESCRIPTION
**Pull request by shunjizhan**
## Change
- this version includes latest addresses for fee and homa factory
- include utils in build so other pkgs can reuse `evmToAddr32`
- some code refactor and lint fix

### CODE REVIEW

File: scripts/utils.ts [hunk 1]
```diff
@@ -1,48 +1,16 @@
...
-    factoryAddr: factory.address,
-    usdcAddr: usdcAddr,
-    feeRegistryAddr: fee.address,
-    routerFee: ethers.utils.formatEther(await fee.getFee(usdcAddr)),
-  });
-  console.log('');
-
-  return { deployer, user, relayer, usdt, fee, factory, ...ADDRESSES[network.name] };
-};
+export type Resolved<T> = T extends Promise<infer U> ? U : T;
```
Comment (xlc) [inline scripts/utils.ts:6]: There is already `Awaited` https://www.typescriptlang.org/docs/handbook/utility-types.html#example

Comment (shunjizhan) [reply scripts/utils.ts]: great to know, obviously I asked old chatgpt who doesn't know this yet lol

Comment (xlc) [reply scripts/utils.ts]: I use phind instead of chatgpt https://www.phind.com/search?cache=shuz8urtzjp2lhxuibfhkgew

Comment (shunjizhan) [reply scripts/utils.ts]: oh this is really cool, will try it out! It even has a vscode extension

**Pull request merged**